npm/vue/test-tsd/shims-vue.d.ts

```diff
@@ -0,0 +1,5 @@
+declare module '*.vue' {
```

Comment:
As the errors shows, `tsc` does not know how to deal with `.vue` files: "Cannot find module './Slots.vue' or its corresponding type declarations." This shim makes it all work as expected.